q2_moshpit/eggnog/_method.py

```diff
@@ -157,3 +158,35 @@ def fetch_eggnog_db() -> EggnogRefDirFmt:
 
     # Return objects
     return eggnog_db
+
+
+def fetch_diamond_db() -> DiamondDatabaseDirFmt:
+    """
+    Downloads diamond reference database using the
+    `download_eggnog_data.py` script from eggNOG. Here, this
+    script downloads 1 file (8.6 Gb).
+    """
+
+    # Initialize output objects
+    diamond_db = DiamondDatabaseDirFmt()
+
+    # Define command.
+    # 'printf "n\nn\ny" will answer the prompts thrown by
+    # download_eggnog_data.py, effectively only downloading the
+    # Diamond DB
+    cmd = [
```

Comment:
We need to break this down a little - there is no need to run that as a single command:
- move the `echo` statements to Python print statements
- break your single call to `subprocess.run` into two: one for `wget` and another one for `gunzip`
- remove the `shell=True` - it's a practice generally advised against: running commands through shell has its own security concerns (https://docs.python.org/3/library/subprocess.html#security-considerations)